app/src/main/java/com/example/nsc_events/screen/RegisterationPage.kt

```diff
@@ -0,0 +1,72 @@
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.*
+import androidx.compose.material3.Button
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun RegistrationPage() {
+    var firstName by remember { mutableStateOf("") }
+    var lastName by remember { mutableStateOf("") }
+    var studentId by remember { mutableStateOf("") }
+    var selectedRole by remember { mutableStateOf("") }
+    var email by remember { mutableStateOf("") }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        TextField(
+            value = firstName,
+            onValueChange = { firstName = it },
+            label = { Text("First Name") }
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        TextField(
+            value = lastName,
+            onValueChange = { lastName = it },
+            label = { Text("Last Name") }
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        TextField(
+            value = studentId,
+            onValueChange = { studentId = it },
+            label = { Text("Student ID Number") },
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number)
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        DropdownMenu(
+            expanded = selectedRole.isNotEmpty(),
+            onDismissRequest = { },
+        ) {
+            DropdownMenuItem(onClick = { selectedRole = "Student" }) {
+                Text("Student")
+            }
+            DropdownMenuItem(onClick = { selectedRole = "Faculty" }) {
+                Text("Faculty")
+            }
+        }
```

Comment:
I'm getting the error No value passed for parameter 'text' for line 53 when I try to run the code.

Reply:
I'm also getting that same error. It's saying there a type mismatch when I hover over the "Text(item)"

Type mismatch.
Required: MutableInteractionSource
Found: () → Unit